web/annotation_editor_layer_builder.css

```
@@ -270,6 +271,25 @@
         margin-inline: 2px;
       }
 
+      .highlightButton {
+        width: var(--editor-toolbar-height);
+
+        &::before {
+          content: "";
+          mask-image: var(--toolbarButton-editorHighlight-icon);
```

Sorry, but this cannot work since that variable isn't available here. (Please keep these examples in mind: https://github.com/mozilla/pdf.js/tree/master/examples/components)

Please add something like this:
```css
--editor-toolbar-highlight-image: url(images/toolbarButton-editorHighlight.svg);
```
*after* this line: https://github.com/mozilla/pdf.js/blob/b1a04fc1848dc7dd3515851c0b34da635e16780a/web/annotation_editor_layer_builder.css#L186
such that you can change this to:
```suggestion
          mask-image: var(--editor-toolbar-highlight-image);
```